NeuroID/NeuroIDClass/Extensions/NIDTesting.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  NIDTesting.swift
+//  NeuroID
+//
+//  Created by Kevin Sites on 3/7/24.
+//
+
+import Foundation
+
+public extension NeuroID {
+    // MARK: Used for Testing Only
+
+    static func setDevTestingURL() {
```

Comment:
This should not be exposed via the public NeuroID interface. The purposed of the `NIDNetworkServiceTestImpl` I implemented was to change this to a dev url for our internal testing.